### ISSUE TITLE
Swiping issue

### ISSUE THREAD

**Issue by kindaPM** [Microflow/Mykra#1]
We tested it in Ximending, and it's a little sticky.

**Comment by Morningstar88** [comment]
Yes.. swipe is not so good right now. I will look at the CSS and JS and try and fix it. 

One solution is to only have 3 projects showcasing per day. This way the user will not be confused with too many projects. 

Coffee meets Bagel is a dating app. They only showed users one profile per day. They raised 20 million and became popular.
